benchmarks/dynamo/common.py

```diff
@@ -2293,12 +2293,21 @@ def warmup(fn, model, example_inputs, mode, niters=5):
             eager_latency, eager_peak_mem, _ = warmup(
                 self.model_iter_fn, model, example_inputs, "eager"
             )
-            optimized_model_iter_fn = optimize_ctx(self.model_iter_fn)
+
+            if self.args.export_aot_inductor:
+                t_0 = time.perf_counter()
+                optimized_model_iter_fn = optimize_ctx(self.model_iter_fn)
+                t_1 = time.perf_counter()
+                additional_time = t_1 - t_0
```

Comment:
Nit: rename `additional_time` to something more meaningful, e.g. `aot_compilation_time`